en/setup/dev_env_windows_cygwin.md

```diff
@@ -1,202 +1,225 @@
-
-Continue at this stage with [the detailed instructions on how to build PX4](../setup/building_px4.md).
 
 ### Windows & Git Special Cases
 
 #### Windows CR+LF vs Unix LF Line Endings
-It is recommended that you force unix style LF endings for every repository you're working with using this Toolcahin and use an editor which preserves them when saving your changes like eclipse or VS Code.
-Compilation of source files also works with CR+LF endings checked out locally but there are cases in Cygwin like execution of shell scripts that require unix line endings otherwise you get errors like `$'\r': Command not found.`.
+
+We recommend that you force Unix style LF endings for every repository you're working with using this toolchain (and use an editor which preserves them when saving your changes - e.g. Eclipse or VS Code).
+Compilation of source files also works with CR+LF endings checked out locally, but there are cases in Cygwin (e.g. execution of shell scripts) that require Unix line endings ( otherwise you get errors like `$'\r': Command not found.`).
 Luckily git can do this for you when you execute the two commands in the root directory of your repo:
 ```
 git config core.autocrlf false
 git config core.eol lf
 ```
-If you work with this Toolchain on multiple repositories you can also set these two configurations globally for your machine by `git config --global ...` but this will apply system wide and hence also in other unrelated git usages on your Windows and is not recommended.
 
-#### Unix permissions execution bit
-Under unix there's a flag in the permissions of each file which tells the OS if the file is allowed to be executed or not. Windows has a different permission system and no such bit but git especially under cygwin supports and cares about that bit.
-This often results in git finding difffereces in permissions even if there is no real diff which looks like this:
+If you work with this toolchain on multiple repositories you can also set these two configurations globally for your machine:
+```
+git config --global ...
+```
+This is not recommended because it may affect any other (unrelated) git use on your Windows machine.
+
+#### Unix Permissions Execution Bit
+
+Under Unix there's a flag in the permissions of each file which tells the OS whether or not the file is allowed to be executed. 
+*git* under Cygwin supports and cares about that bit (even though Windows has a different permission system). 
+This often results in *git* finding differences in permissions even if there is no real diff which looks like this:
 ```
 diff --git ...
 old mode 100644
 new mode 100755
 ```
 
-It's recommended to disable this functionality by executing `git config core.fileMode false` in every repo you use with this toolchain.
+We recommend disabling this functionality by executing `git config core.fileMode false` in every repo where you use with this toolchain.
+
+## Shell Script Installation {#script_setup} 
```

Comment:
@MaEtUgR  It isn't clear to me what the Shell Script vs Manual Installation are for. Ie when you would use them. This should be stated. I'm pretty sure that the manual section is how to extend the toolchain, so it should say that, and possibly cover how you build/make the installer package.